inc/settings-helpers.php

```diff
@@ -69,3 +71,32 @@ function ang_settings_get_option( $option_name, $default = '' ) {
 
 	return Admin_Settings::get_option( $option_name, $default );
 }
+
+
+/**
+ * Update Elementor Kit Option with respect to GSK.
+ *
+ * @return void
+ */
+function ang_update_elementor_kit() {
+	if ( empty( $_POST ) ) { // phpcs:ignore
+		return;
+	}
+
+	$data              = $_POST; // phpcs:ignore
+	$key               = 'global_kit';
+	$elementor_kit_key = \Elementor\Core\Kits\Manager::OPTION_ACTIVE;
+	$kit_id            = Options::get_instance()->get( 'global_kit' );
+
+	$raw_value     = isset( $data[ $key ] ) ? wp_unslash( $data[ $key ] ) : $kit_id;
+	$elementor_kit = \get_option( $elementor_kit_key );
+
+	if ( $raw_value !== $kit_id || $raw_value !== $elementor_kit ) {
+		if ( empty( $raw_value ) || '-1' === $raw_value ) {
+			\update_option( $elementor_kit_key, false );
```

Comment:
Assuming this is the scenario where you leave Global Kit to be blank?

If so, Elementor Global Kit can't be false. Instead, we need to revert it to original Default Kit as intended by Elementor. I have it saved in our option in `default_kit` key during migration which we could use.

```suggestion
			Options::get_instance()->get( 'default_kit' );
```